handbook/ceo/index.md

```diff
@@ -13,3 +13,12 @@ The best way to communicate is [asynchronously and publicly](../communication.md
 My preferred working hours are 10:30-17:00 PT and 20:00-03:00 PT (Google Calendar does not let me specify a noncontigous "working hours" block).
 
 I've found recurring meetings with specific single teammates (other than my [1-1s](../leadership/1-1.md)) to be ineffective. In most cases, people either want to discuss topics with me that are best discussed with a larger group or someone else, or they don't have any topics to discuss but don't want to offend me by canceling the recurring event.
+
+### Asking for a review from me
+
+If you need me to review something (such as a blog post or plan), please state:
+
+- The context (ideally a single link to an RFC or other written down doc)
+- What input you need from me
+
+This is to avoid me jumping in and making low-level wording fixes when you're looking for high-level strategy input.
```

Comment:
```suggestion
This is to avoid me jumping in and making low-level wording fixes when you're looking for high-level strategy input.

If the document to be reviewed isn't private, please post the review request with the context above [publicly](../communication.md). Other people involved may have relevant context of their own to share.
```